src/data/data-files/azure-regions.json

```diff
@@ -0,0 +1,378 @@
+[
+    {
+      "Latitude": "37.3719",
+      "Longitude": "-79.8164",
+      "RegionName": "eastus"
+    },
+    {
+      "Latitude": "36.6681",
+      "Longitude": "-78.3889",
+      "RegionName": "eastus2"
+    },
+    {
+      "Latitude": "29.4167",
+      "Longitude": "-98.5",
+      "RegionName": "southcentralus"
+    },
+    {
+      "Latitude": "47.233",
+      "Longitude": "-119.852",
+      "RegionName": "westus2"
+    },
+    {
+      "Latitude": "33.448376",
+      "Longitude": "-112.074036",
+      "RegionName": "westus3"
+    },
+    {
+      "Latitude": "-33.86",
+      "Longitude": "151.2094",
+      "RegionName": "australiaeast"
+    },
+    {
+      "Latitude": "1.283",
+      "Longitude": "103.833",
+      "RegionName": "southeastasia"
+    },
+    {
+      "Latitude": "53.3478",
+      "Longitude": "-6.2597",
+      "RegionName": "northeurope"
+    },
+    {
+      "Latitude": "60.67488",
+      "Longitude": "17.14127",
+      "RegionName": "swedencentral"
+    },
+    {
+      "Latitude": "50.941",
+      "Longitude": "-0.799",
+      "RegionName": "uksouth"
+    },
+    {
+      "Latitude": "52.3667",
+      "Longitude": "4.9",
+      "RegionName": "westeurope"
+    },
+    {
+      "Latitude": "41.5908",
+      "Longitude": "-93.6208",
+      "RegionName": "centralus"
+    },
+    {
+      "Latitude": "-25.731340",
+      "Longitude": "28.218370",
+      "RegionName": "southafricanorth"
+    },
+    {
+      "Latitude": "18.5822",
+      "Longitude": "73.9197",
+      "RegionName": "centralindia"
+    },
+    {
+      "Latitude": "22.267",
+      "Longitude": "114.188",
+      "RegionName": "eastasia"
+    },
+    {
+      "Latitude": "35.68",
+      "Longitude": "139.77",
+      "RegionName": "japaneast"
+    },
+    {
+      "Latitude": "37.5665",
+      "Longitude": "126.9780",
+      "RegionName": "koreacentral"
+    },
+    {
+      "Latitude": "43.653",
+      "Longitude": "-79.383",
+      "RegionName": "canadacentral"
+    },
+    {
+      "Latitude": "46.3772",
+      "Longitude": "2.3730",
+      "RegionName": "francecentral"
+    },
+    {
+      "Latitude": "50.110924",
+      "Longitude": "8.682127",
+      "RegionName": "germanywestcentral"
+    },
+    {
+      "Latitude": "59.913868",
+      "Longitude": "10.752245",
+      "RegionName": "norwayeast"
+    },
+    {
+      "Latitude": "-23.55",
+      "Longitude": "-46.633",
+      "RegionName": "brazilsouth"
+    },
+    {
+      "Latitude": "36.6681",
+      "Longitude": "-78.3889",
+      "RegionName": "eastus2euap"
+    },
+    {
+      "Latitude": null,
```

Comment:
These `null` regions are pretty much useless, and may be related to particular MS access we have.  We should remove them

Reply:
> These `null` regions are pretty much useless, and may be related to particular MS access we have. We should remove them

hmm. This file is interesting. Before moving it to a new folder, it had 'weird' chars and the app crashed. I'll do the cleanup of this too.